src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -653,109 +653,79 @@ private void DoParse(string configuration, bool ignoreUnknown)
 
             Clear();
 
-            // break it down by commas
-            var arr = configuration.Split(StringSplits.Comma);
-            Dictionary<string, string> map = null;
-            foreach (var paddedOption in arr)
+            if (configuration.TrimStart().StartsWith("redis://", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
or `rediss://`, which should enable TLS

Reply:
fixed

Reply:
In case of `rediss://` it will set `Ssl = true` and `SslProtocols = Tls12 | Tls13`. Is it OK?